index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <section class="section">
+    <div id="container" class="container">
+      <div class="imageContainer">
```

Comment:
@ashok-choudhary-0 Can you add images from your script file?
So there will be an array of images and then the function will render it.